documentation/bolt_configuration_reference.md.erb

```diff
@@ -80,6 +80,18 @@ plugin_hooks:
       message: "Which version of Puppet do you want to install?"
 ```
 
+## Project configuration options
+
+> **Note:** Project configuration is experimental
+
+Bolt projects are configured in `Boltdir/project.yaml`, and can define the following settings:
```

Comment:
```suggestion
Use `<PROJECT_NAME>/project.yaml` to whitelist the tasks or plans that are displayed by the `bolt [plan|task] show` command. You can define the following settings:
```

Reply:
I know this narrows the focus a bit, but we can always go back and edit this if/when we expand the project.yaml functionality.